meta-facebook/grand-teton/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(grand-teton)
```

Comment:
gt-cc

Reply:
We change the project name to gt-cc, please review it again. Thanks.